sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.FormRecognizer
+{
+    /// <summary>
+    /// </summary>
+    public class FormRecognizerClient
+    {
+        private readonly ServiceClient _serviceClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FormRecognizerClient"/>.
+        /// </summary>
+        public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential) : this(endpoint, credential, new FormRecognizerClientOptions())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FormRecognizerClient"/>.
+        /// </summary>
+        public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential, FormRecognizerClientOptions options)
+        {
+            var diagnostics = new ClientDiagnostics(options);
+            var pipeline = HttpPipelineBuilder.Build(options, new AzureKeyCredentialPolicy(credential, Constants.AuthorizationHeader));
+            _serviceClient = new ServiceClient(diagnostics, pipeline, endpoint.ToString());
+        }
+
+        /// <summary>
+        /// </summary>
+        protected FormRecognizerClient()
+        {
+        }
+
+        #region Content
+
+        /// <summary>
+        /// Extracts layout elements from one or more passed-in forms.
+        /// </summary>
+        /// <param name="formFileStream">The stream containing one or more forms to extract elements from.</param>
+        /// <param name="contentType">The content type of the input file.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A Operation&lt;IReadOnlyList&lt;ExtractedLayoutPage&gt;&gt; to wait on this long-running operation.  Its Operation&lt;IReadOnlyList&lt;ExtractedLayoutPage&gt;&gt;.Value upon successful
+        /// completion will contain layout elements extracted from the form.</returns>
+        [ForwardsClientCalls]
```

Comment:
Ohh cool! I had no idea this was a thing!

Reply:
Are you talking about `ForwardsClientCalls`? It's just a workaround for tests, but we intend to remove it: #11023. It's not something we need for the `CustomForm` methods now since we don't have tests for them yet.